nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -70,6 +70,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [Kea](https://www.isc.org/kea/), ISCs 2nd generation DHCP and DDNS server suite. Available at [services.kea](options.html#opt-services.kea).
 
+- [mozillavpn](https://github.com/mozilla-mobile/mozilla-vpn-client), the client for the [Mozilla VPN](https://vpn.mozilla.org/) service. Available as [services.mozillavpn](options.html#opt-services.mozillavpn).
+
```

Comment:
Maybe you want to put this in `nixos/doc/manual/release-notes/rl-2205.section.md`?

Reply:
Fixed, thanks.